x-pack/plugins/monitoring_collection/server/routes/rules.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Observable } from 'rxjs';
+import { IRouter, ServiceStatus } from '../../../../../src/core/server';
+import { getESClusterUuid, getKibanaStats } from '../lib';
+import { MetricResult } from '../plugin';
+
+export function registerRulesRoute({
+  router,
+  config,
+  overallStatus$,
+  getMetrics,
+}: {
+  router: IRouter;
+  config: {
+    allowAnonymous: boolean;
+    kibanaIndex: string;
+    kibanaVersion: string;
+    uuid: string;
+    server: {
+      name: string;
+      hostname: string;
+      port: number;
+    };
+  };
+  overallStatus$: Observable<ServiceStatus>;
+  getMetrics: () => Promise<MetricResult[] | undefined>;
+}) {
+  router.get(
+    {
+      path: '/api/monitoring_collection/rules',
+      options: {
+        // authRequired: !config.allowAnonymous,
+        tags: ['api'], // ensures that unauthenticated calls receive a 401 rather than a 302 redirect to login page
+      },
+      validate: false,
+    },
+    async (context, req, res) => {
+      const [rules, clusterUuid, kibana] = await Promise.all([
```

Comment:
This code is pretty generic and I'm not married to this idea of a specific route per type here - I think we could also go two other ways:

1) The plugin that authors the metrics can expose it's own HTTP endpoint for the data
2) We can expose a single HTTP endpoint with parameters that indicate what type of data to return